Time remaining component

rajbirjohar opened the pull request:
Component written. Message "time remaining" on top is missing and having trouble generating with other components. Counter not working yet.

rajbirjohar committed:
Countdown-Component

Added TIme-Remaining Component including the box.

rajbirjohar committed:
Attempted to to render time-remaining component

jshin029 commented:
Could you fix the merge conflicts here? Also don't render the countdown component on home.js you should render it on live.js. Not sure if you use this but take a look at this package https://www.npmjs.com/package/react-countdown-now

jshin029 commented:
Also there are some merge conflicts can you fix those too

rajbirjohar committed:
Imported Countdown-package

Attempting to redo the countdown timer

rajbirjohar committed:
Update TimeRemaining.js

rajbirjohar committed:
Merge branch 'master' into time-remaining-component

rajbirjohar committed:
Countdown rendered

Countdown has been rendered but styling still needs to be written as well as "time remaining"

rajbirjohar committed:
Styled Timer

Was able to render all required text in the format sufficiently close to figma. Position needs work as it will only stay on the top left corner and is not responsive.

rajbirjohar committed:
Positioning

Fixed Position. Only descrepancy is the bold styling. I cannot seem to replicate it no matter how many ways I style it.

rajbirjohar committed:
Update TimeRemaining.css
